### ISSUE TITLE
dbname.padEnd(20) is not a function

### ISSUE THREAD

**Issue by danifitz** [glynnbird/couchreplicate#17]
```
couchreplicate -v
> 0.2.1
```
When  I invoke `couchreplicate` I get an error:
```
TypeError: dbname.padEnd is not a function
    at migrateDB (/usr/local/lib/node_modules/couchreplicate/index.js:191:28)
    at async.queue (/usr/local/lib/node_modules/couchreplicate/index.js:265:7)
    at /usr/local/lib/node_modules/couchreplicate/node_modules/async/dist/async.js:4082:9
    at Object.q.process (/usr/local/lib/node_modules/couchreplicate/node_modules/async/dist/async.js:2330:17)
    at /usr/local/lib/node_modules/couchreplicate/node_modules/async/dist/async.js:2238:19
    at Immediate.<anonymous> (/usr/local/lib/node_modules/couchreplicate/node_modules/async/dist/async.js:119:16)
    at runCallback (timers.js:574:20)
    at tryOnImmediate (timers.js:554:5)
    at processImmediate [as _immediateCallback] (timers.js:533:5)
```

I am running the following command:

```
couchreplicate -a -s https://<user>:<password>@<host>.cloudant.com -t https://<user>:<password>@<host>.cloudant.com
```

I've installed `0.2.0` and that also gives me the same error. Passing the --quiet flag in `0.2.1` prevents the error.

**Comment by glynnbird** [comment]
Thanks for spotting this @danifitz. I'll fix

**Comment by glynnbird** [comment]
Fixed in 0.2.2. the `String.padEnd` function is relatively new. I implemented my own for wider compatibility.
